snippets/core/system-text-json/csharp/RoundtripLongToString.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Text.Json;
+
+namespace SystemTextJsonSamples
+{
+    public class RoundtripLongToString
+    {
+        public static void Run()
+        {
+            string jsonString;
+
+            // Serialize to create input JSON
+            var weatherForecast = WeatherForecastFactories.CreateWeatherForecastWithLong();
```

Comment:
nit: If we were to follow dotnet/runtime guidelines, don't use `var` here since it isn't clear from R.H.S. what the type actually is.